datahub-web-react/src/app/search/AdvancedSearchAddFilterSelect.tsx

```diff
@@ -0,0 +1,50 @@
+import { Select } from 'antd';
+import * as React from 'react';
+import styled from 'styled-components';
+import { PlusOutlined } from '@ant-design/icons';
+
+import { FacetFilterInput } from '../../types.generated';
+import { FIELD_TO_LABEL } from './utils/constants';
+
+const StyledPlus = styled(PlusOutlined)`
+    margin-right: 6px;
+`;
+
+interface Props {
+    selectedFilters: Array<FacetFilterInput>;
+    onFilterFieldSelect: (value) => void;
+}
+
+const { Option } = Select;
+
+export const AdvancedSearchAddFilterSelect = ({ selectedFilters, onFilterFieldSelect }: Props) => {
+    return (
+        <Select
+            value={{
+                value: 'value',
+                label: (
+                    <div>
+                        <StyledPlus />
+                        Add Filter
+                    </div>
+                ),
+            }}
+            labelInValue
+            style={{ padding: 6, fontWeight: 500 }}
```

Comment:
I know we're trying to use styled components across our React app, can we do that here instead?

Reply:
ach I wanted to do that here too but the antd `Select` was overwriting them